pkg/apis/starrocks/v1/load_type.go

```diff
@@ -67,10 +67,6 @@ type StarRocksLoadSpec struct {
 	// +optional
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
 
-	// Probe defines the mode probe service in container is alive.
```

Comment:
is this forgot to remove in last startuptime period PR?

Reply:
yes